vhdl/autogen_quartus.py

```diff
@@ -26,3 +32,11 @@ def write_tcl(input_struct, outputFilename):
         out_file.write(create_mm_connection_point(input_struct))
         out_file.write(create_sink_connection_point(input_struct))
         out_file.write(create_source_connection_point(input_struct))
+
+def get_working_dir(dir):
+    if not(dir.endswith("\\")):
+        dir += "\\"
+    return dir + "quartus\\"
+        
+if __name__ == "__main__":
+    main('MNR_dataplane.json', "C:\\Users\\wickh\\Documents\\NIH\\simulink_models\\models\\short_window_mean_reduction\\hdlsrc\\MNR\\MNR_dataplane_avalon_hw.tcl", "C:\\Users\\wickh\\Documents\\NIH\\simulink_models\\models\\short_window_mean_reduction\\hdlsrc\\MNR")
```

Comment:
not a big deal right now, but when we start refactoring our code base, we should make this script a command-line executable as well, e.g. use `argparse` and follow the usual pattern:
```python
if __name__ == "__main__":
    args = parseargs()
    main(args)
```

Reply:
by "not a big deal", I mostly mean I think it's more important to get the dtogen stuff in a "finished" state right now instead of doing this.

Reply:
I'm going to disagree with you here.  :)  I like the idea of using the `argparse`, but we should also make this change now.  We should take the extra time now to design the code how we want it to run now, then the final review should go more smoothly.

Reply:
Perfectly valid point :smile: 

Here are some examples of how we've used `argparse` elsewhere in our codebase:
- [combine_linker_configs.py](https://github.com/fpga-open-speech-tools/simulink_codegen/blob/master/ui/combine_linker_configs.py)
- [combine_ui_configs.py](https://github.com/fpga-open-speech-tools/simulink_codegen/blob/master/ui/combine_ui_configs.py)
- [gen_makefile.py](https://github.com/fpga-open-speech-tools/simulink_codegen/blob/master/device_drivers/gen_makefile.py)
- [awsdownloader.py](https://github.com/fpga-open-speech-tools/utils/blob/aws/runtime_config/awsdownloader.py)

Reply:
Completed. Feel free to review the implementations. Creation of the Platform Designer component has been consolidated and moved to `create_hw_tcl.py` that is also available for use from the command line.